public/app/features/alerting/unified/components/rule-editor/QueryRows.tsx

```diff
@@ -249,6 +252,12 @@ function newModel(item: AlertQuery, settings: DataSourceInstanceSettings): Omit<
       instant: isInstant,
     },
   };
+
+  if (isInstant) {
+    newQuery.model.instant = isInstant;
+  }
```

Comment:
Why do we have this, if `newModel` is initialised with `newQuery.model.instant: isInstant` anyway? Or am I missing something?